pyface/resource/resource_manager.py

```diff
@@ -13,7 +13,7 @@
 sounds etc.
 """
 
-import collections.abc, glob, inspect, os, sys, types
+import collections.abc, glob, inspect, os, types
```

Comment:
These imports should be converted to PEP8

Reply:
This is an `E401` flake8 failure and im hoping to address those separately.

Reply:
Fair enough